### PR TITLE
UIREQ-722: Add success toast to Requests

### PR DESCRIPTION
**Pull request by pavelspichonak**
## Purpose
Add success toast to Requests

## Refs
https://issues.folio.org/browse/UIREQ-722

## Notice
One test was skipped. We checked that the functionality covered by this test works fine. We think that test fails by timeout. Investigation of this problem was added at spike at the next sprint.

### CODE REVIEW

File: src/routes/RequestsRoute.js [hunk 1]
```diff
@@ -657,9 +657,28 @@ class RequestsRoute extends React.Component {
...
+    const query = new URLSearchParams(this.props.location.search);
+    const mode = query.get('mode');
+
     return this.props.mutator.records.POST(data)
-      .then(() => this.closeLayer())
-      .catch(resp => this.processError(resp));
+      .then(() => {
+        this.closeLayer();
+
+        const message = mode === createModes.DUPLICATE
```
Comment (pavelspichonak) [inline src/routes/RequestsRoute.js:667]: => isDuplicateMode function + tests

**Review by Dmitriy-Litvinenko — APPROVED** [review]

File: src/routes/RequestsRoute.js [hunk 2]
```diff
@@ -657,9 +657,28 @@ class RequestsRoute extends React.Component {
...
+        this.context.sendCallout({ message });
+      })
+      .catch(resp => {
+        this.processError(resp);
+
+        const message = mode === createModes.DUPLICATE
+          ? <FormattedMessage id="ui-requests.duplicateRequest.fail" />
+          : <FormattedMessage id="ui-requests.createRequest.fail" />;
+
+        this.context.sendCallout({ message });
```
Comment (pavelspichonak) [inline src/routes/RequestsRoute.js:680]: type error

File: src/ViewRequest.js [hunk 3]
```diff
@@ -220,6 +220,13 @@ class ViewRequest extends React.Component {
 
     this.props.mutator.selectedRequest.PUT(updatedRecord).then(() => {
       this.props.onCloseEdit();
+      this.callout.current.sendCallout({
+        message: <FormattedMessage id="ui-requests.editRequest.success" />,
+      });
+    }).catch(() => {
+      this.callout.current.sendCallout({
+        message: <FormattedMessage id="ui-requests.editRequest.fail" />,
```
Comment (pavelspichonak) [inline src/ViewRequest.js:228]: type error

**Review by Stanislau-Snapok — APPROVED** [review]

**Comment by zburke** [comment]
@Stanislau-Snapok , @pavelspichonak had labeled this "do not merge" but you just merged it. If that was deliberate, please remove the label; if it was a mistake, please revert :)

**Comment by Stanislau-Snapok** [comment]
> 

It was deliberate, just forgot to remove the label ;)

**Pull request merged**